GenericsHomework/GenericsHomework/Node.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace GenericsHomework
+{
+    public class Node<T>
+    {
+        public Node(T init) 
+        {
+            this.data = init;
+            this.Next = this;
+        }
+        
+        public Node()
+        {
+            this.Next = this;
+        }
+
+        public T? data;

```

Comment:
```suggestion
        public T _Data;
```
This does not need to be marked as nullable.